src/domain/core/src/auth/dataset_action_authorizer.rs

```diff
@@ -23,13 +23,14 @@ use crate::AccessError;
 pub trait DatasetActionAuthorizer: Sync + Send {
     async fn check_action_allowed(
         &self,
-        dataset_handle: &DatasetHandle,
+        // TODO: Private Datasets: just odf::DatasetID? here and below
```

Comment:
Absolutely. Previously we queried the dataset repository, now we don't

Reply:
To be implemented in a follow-up PR